app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -69,4 +71,11 @@ class BindingTariffClassificationConnector @Inject()(configuration: AppConfig, c
     client.GET[Seq[Event]](url)
   }
 
+  def search(search: Search, sort: Sort)(implicit hc: HeaderCarrier, binder: QueryStringBindable[String]): Future[Seq[Case]] = {
+    val searchString = if(!search.isEmpty) "&" + Search.bindable.unbind("", search) else ""
```

Comment:
Are there any helpers in Play for building query strings so that we don't have to concat the & ourselves?

Reply:
The search string can already contain many `&`s though, I doubt there will be a library that lets us do that

Reply:
Libraries would end up encoding the &

Reply:
👍